src/io/output_time.cc

```diff
@@ -114,7 +118,7 @@ Input::Iterator<Input::Record> OutputTime::get_output_mesh_record() {
 
 std::shared_ptr<OutputMeshBase> OutputTime::create_output_mesh_ptr(bool init_input) {
 	bool discont = (used_interpolations_.find(DiscreteSpace::CORNER_DATA) != used_interpolations_.end());
-	if (discont) {
+	if (discont || this->get_output_mesh_record()) {
```

Comment:
I did this, so that the discontinuous output mesh is created when OutputMesh record is specified (i has only keys defining refinement).